src/main/kotlin/ch/unibas/dmi/dbis/cottontail/model/values/complex/ComplexArray.kt

```diff
@@ -0,0 +1,49 @@
+package ch.unibas.dmi.dbis.cottontail.model.values.complex
+
+/**
+ * An array of complex numbers.
+ * @constructor Creates a new array of the specified [size], with all elements initialized to zero.
+ *
+ * @author Manuel Huerbin
+ * @version 1.0
+ */
+class ComplexArray(size: Int) {
```

Comment:
I don't quite see what this class is needed for; why don't you just implement ComplexVectorValue as Value<Array<Complex>>?